nss-day-cohort-60/rare-client-billowing-smurfs
# Jl add post

### Add POST request to /posts

## To Test:
1. pull jl-add-post branch
2. checkout to jl-add-post branch
3. navigate to the post form using the navbar "New Post" link
4. fill out the form and submit
5. you should navigate to the post list page with your new post at the end of the list.
